.github/CONTRIBUTING.md

```diff
@@ -5,6 +5,36 @@ appreciated and welcomed. If you're planning a new feature or changing
 the API, please create an issue first. This way we can ensure that your precious
 work is not in vain.
 
+Table of Contents
+
+- [Issues](#issues)
+- [Your first Contribution](#your-first-contribution)
+- [Setup](#setup)
+    - [Using npm (setup)](#using-npm-setup)
```

Comment:
I would remove "(setup)" & "(testing)", I mean changing
- [Using npm (setup)](#using-npm-setup)
to
- [Using npm](#using-npm-setup)